### PR TITLE
Prepare for next version

### PR DESCRIPTION
**Pull request by leandroberetta**

### CODE REVIEW

**Review by aljesusg — APPROVED** [review]
LGFM , only one comment related with dates of v1.53 I don't know why this date has been changed

File: manifests/kiali-community/1.53.0/manifests/kiali.v1.53.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ metadata:
     support: Kiali
     description: "Kiali project provides answers to the questions: What microservices are part of my Istio service mesh and how are they connected?"
     repository: https://github.com/kiali/kiali
-    createdAt: 2022-06-13T12:27:23Z
+    createdAt: 2022-07-04T07:22:24Z
     alm-examples: |-
```
[the remark this answers is not in the document]
Comment (jmazzitelli) [reply manifests/kiali-community/1.53.0/manifests/kiali.v1.53.0.clusterserviceversion.yaml:16]: We added this feature to the CI workflow last sprint. This is what we want. Today is 07-04 and is when this 1.53 release has been published. Thus we want the CSV to reflect this date. Notice the old date of June 13 is meaningless for 1.53 - that isn't the date 1.53 was created. Today (July 4) is when 1.53 was created/published so we want that date.

File: manifests/kiali-upstream/1.53.0/manifests/kiali.v1.53.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -12,7 +12,7 @@ metadata:
     support: Kiali
     description: "Kiali project provides answers to the questions: What microservices are part of my Istio service mesh and how are they connected?"
     repository: https://github.com/kiali/kiali
-    createdAt: 2022-06-13T12:27:22Z
+    createdAt: 2022-07-04T07:22:24Z
     alm-examples: |-
```
[the remark this answers is not in the document]
Comment (jmazzitelli) [reply manifests/kiali-upstream/1.53.0/manifests/kiali.v1.53.0.clusterserviceversion.yaml:16]: The same answer as above.

File: manifests/kiali-community/1.53.0/manifests/kiali.v1.53.0.clusterserviceversion.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (aljesusg) [reply manifests/kiali-community/1.53.0/manifests/kiali.v1.53.0.clusterserviceversion.yaml:16]: I assumed something like that thanks for clarifying @jmazzitelli

**Pull request merged**